lit_tests/kernel/wave/expansion.py

```diff
@@ -85,21 +85,21 @@ def test_read_write_equal_sizes():
         # CHECK: %a
         # CHECK-NEXT: %c
         # CHECK-NEXT: %read_0_0
```

Comment:
Can we avoid printing the mapping and shape when it is None?

Reply:
Yeah, good idea.

Reply:
I can update `CustomOp` printing, but I can't do anything with `%read_3 : [num_users=1] = call_function[target=shark_turbine.kernel.ops.wave_ops.read](args = (%allocate_1, 4, None, None), kwargs = {})` form.

Reply:
https://github.com/iree-org/iree-turbine/pull/64

Reply:
Thanks!